bladeui/js/html/resettedPassword.js

```diff
@@ -0,0 +1,14 @@
+"use strict";
+
+const header = require("./common/header");
+
+const html = `
+<div class="recover-completed-view">
+  ${header(`New password successfully saved, 
+  you can continue earning ADB tokens as you browse the web!`)}
```

Comment:
maybe, it will be better with the message `you can continue earning ADB tokens when you browse the web` or `you can continue earning ADB tokens browsing the web`?

Reply:
this text from Tylor`s design, dont think we should change it